resources/eventing/charts/nats/values.yaml

```diff
@@ -13,13 +13,83 @@ nats:
     leafnodes: 7422
     gateways: 7522
 
+  # The servers name prefix, must be used for example when we want a NATS cluster
+  # spanning multiple Kubernetes clusters.
+  serverNamePrefix: ""
+
+  # Server Tags
+  serverTags:
+  # - "foo"
+  # - "bar"
+
```

Comment:
If `serverNamePrefix` and `serverTags` are only useful when we want a NATS cluster spanning multiple Kubernetes clusters, then we can remove these configs because it not of any use in our scenario. Let's keep only the stuff in the helm chart which is useful for us.